Documentation.md

```diff
@@ -0,0 +1,207 @@
+
+### Modmail
+
+There exists a category (configurable in database) with the purpose of modmail. When a user dms the bot, a channel in this category the modmail channels will be created. There also exists another separate channel for modmaillog, in which the messages will get stored when the thread has been closed.\
+The user will get a response, that the inquiry is being handled and moderators get a notification in modqueue.\
+The following commands can only be used within an existing modmail thread and will execute actions in that modmail-thread.\
+Staff members can then answer the thread by executing the `;reply` command. This command will then pass on the message to the user and log the message in the current thread as well. This message will be in the form of an embed, with the author being marked.   There also exists the functionality of `;anonreply` for which the author will be replaced with a OP logo.\
+The user can then answer in his DM and this message will be passed along to this particular modmail thread and the user will get a reaction, that his message has been processed.\
+This conversation can then go on. In case a staff member wants to get pinged in case the user answers, `;subscribe` can be used. If this should be disabled again `;unsubscribe` serves that purpose.\
+If a staff member wants to delete/edit a message `;edit <messageId> <new Text>` and `;delete <messageId>` serve that purpose. This will edit/delete the message in the thread and also update it in the DM with the user.\
+When the conversation is over a staff member can then use the `;close <note>` command, which will close the thread and log all  the messages between staff members and the users into the modmaillog channel. The actual thread channel will be deleted.\
+There also exists the possibility to use `;disableThread <duration> <note>`. This will also close the thread and make the user unable to contact modmail for the duration set.\
+In case the user tries to contact modmail, he will get an answer at which point in time, the user will be able to contact modmail again.
+
+The following commands are available globally.\
+Modmail for a specific user can be enabled again with `;enableModmail <user>`. After modmail get enabled for a user, he will l only get the notification about when he can contact again once.  Modmail can also be disabled for a specific user via `;disableModmail <user>`. If a staff member wants to contact a user directly, without waiting for the user to write `;contact <user>` serves this purpose. In case there is already a thread going on, an embed will be posted containing a link.
+
+Current implementation only supports one image attachment per message.
+    
+### Ban
+
+`;ban @Username#1234 reason`
+
+Ban user from server, reason parameter is optional.\
+Command can also be used with [user IDs](https://dis.gd/userid) to ban users not present in server with `;banid` command.\
+It is made so that other bots can't be banned with it.
```

Comment:
Add notice, that this ban will get logged into the modlog channel (currently open PR for that tho)